src/main/python/main.py

```diff
@@ -0,0 +1,1123 @@
+import ast
+import inspect
+from de.fraunhofer.aisec.cpg.graph import Annotation
+from de.fraunhofer.aisec.cpg.graph import AnnotationMember
+from de.fraunhofer.aisec.cpg.graph.declarations import Declaration
+from de.fraunhofer.aisec.cpg.graph.declarations import FunctionDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import IncludeDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import MethodDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import NamespaceDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import ParamVariableDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import RecordDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import TranslationUnitDeclaration
+from de.fraunhofer.aisec.cpg.graph.declarations import VariableDeclaration
+from de.fraunhofer.aisec.cpg.graph.statements import CompoundStatement, TryStatement
+from de.fraunhofer.aisec.cpg.graph.statements import DeclarationStatement
+from de.fraunhofer.aisec.cpg.graph.statements import EmptyStatement
+from de.fraunhofer.aisec.cpg.graph.statements import ForEachStatement
+from de.fraunhofer.aisec.cpg.graph.statements import IfStatement
+from de.fraunhofer.aisec.cpg.graph.statements import ReturnStatement
+from de.fraunhofer.aisec.cpg.graph.statements import WhileStatement
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import ArrayRangeExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import ArraySubscriptionExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import BinaryOperator
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import CallExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import ConstructExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import DeclaredReferenceExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import Expression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import Literal
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import MemberCallExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import MemberExpression
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import UnaryOperator
+from de.fraunhofer.aisec.cpg.graph.statements.expressions import InitializerListExpression
+from de.fraunhofer.aisec.cpg.graph.types import TypeParser
+from de.fraunhofer.aisec.cpg.sarif import PhysicalLocation
+from de.fraunhofer.aisec.cpg.sarif import Region
+from java.net import URI
+from java.util import ArrayList
+
+
+#############################
+# PROBLEMS / OPEN QUESTIONS #
+#############################
+# 1. Wie gehen wir mit Expression außerhalb von Funktionen um? Für Funktionen gibt es die FunctionDeclaration. Was gibt es für Expressions?
```

Comment:
Cleanup open issue list, i.e. make it a bit more presentable

Reply:
Resolved in 5ec26c85f37b9d3fda77d70d502110e9e53196a4